src/data/missions.csv

```diff
@@ -934,202 +934,202 @@
+"<size=25>② 【3日目】チーム競技場でファン数を累計1000人以上獲得しよう","<size=25>②  [Day 3] Gather a total of 1,000 fans from the team stadium</size>"
+"<size=25>② 【3日目】デイリーレースに3回勝利しよう","<size=25>②  [Day 3] Win 3 daily races</size>"
+"<size=25>② 【3日目】チーム競技場のマイルのレースに3回勝利しよう","<size=25>②  [Day 3] Win 3 mile races in the team stadium</size>"
+"<size=25>② 【3日目】デイリーレースに[晴]の状態で出走しよう","<size=25>②  [Day 3] Participate in a daily race with clear weather</size>"
+"<size=25>② 【全達成報酬】3日目の限定ミッションを全てクリアしよう","<size=25>②  [Series] Complete all Day 3 missions</size>"
+"<size=25>② 【4日目】チーム編成を行いチームランク[E2]以上に到達させよう","<size=25>②  [Day 4] Put together a PVP team with team rank of at least E2</size>"
+"<size=25>② 【4日目】チーム競技場でウイニング報酬を3回獲得しよう","<size=25>②  [Day 4] Get a winning reward 3 times in the team stadium</size>"
+"<size=25>② 【4日目】チーム競技場の中距離のレースに3回勝利しよう","<size=25>②  [Day 4] Win 3 medium-distance races in the team stadium</size>"
+"<size=25>② 【4日目】デイリーレースに[雨]の状態で出走しよう","<size=25>②  [Day 4] Participate in a daily race with rainy weather</size>"
+"<size=25>② 【全達成報酬】4日目の限定ミッションを全てクリアしよう","<size=25>②  [Series] Complete all Day 4 missions</size>"
+"<size=25>② 【5日目】チーム競技場でスコアを累計50万以上獲得しよう","<size=25>②  [Day 5] Get a total score of 500,000 points or more in the team stadium</size>"
+"<size=25>② 【5日目】デイリーレースに3回勝利しよう","<size=25>②  [Day 5] Win 3 daily races</size>"
+"<size=25>② 【5日目】チーム競技場の長距離のレースに3回勝利しよう","<size=25>②  [Day 5] Win 3 long-distance races in the team stadium</size>"
+"<size=25>② 【5日目】デイリーレースに[内枠]（1～3枠）の状態で出走しよう","<size=25>②  [Day 5] Participate in a daily race with an inner waku</size>"
+"<size=25>② 【全達成報酬】5日目の限定ミッションを全てクリアしよう","<size=25>②  [Series] Complete all Day 5 missions</size>"
+"<size=25>② 【6日目】チーム競技場でファン数を累計2000人以上獲得しよう","<size=25>②  [Day 6] Gather a total of 2,000 fans from the team stadium</size>"
+"<size=25>② 【6日目】チーム競技場で親愛度を累計30上げよう","<size=25>②  [Day 6] Raise the affection level of girls in your team 30 times in the team stadium</size>"
+"<size=25>② 【6日目】チーム競技場のダートレースに3回勝利しよう","<size=25>②  [Day 6] Win 3 dirt races in the team stadium</size>"
+"<size=25>② 【6日目】デイリーレースに[外枠]（6～8枠）の状態で出走しよう","<size=25>②  [Day 6] Participate in a daily race with an outer waku</size>"
+"<size=25>② 【全達成報酬】6日目の限定ミッションを全てクリアしよう","<size=25>②  [Series] Complete all Day 6 missions</size>"
+"<size=25>② 【7日目】チーム編成を行いチームランク[E3]以上に到達させよう","<size=25>②  [Day 7] Put together a PVP team with team rank of at least E3</size>"
+"<size=25>② 【7日目】デイリーレースに3回勝利しよう","<size=25>②  [Day 7] Win 3 daily races</size>"
+"<size=25>② 【7日目】チーム競技場でスコアを累計100万以上獲得しよう","<size=25>②  [Day 7] Get a total score of 1 million points or more in the team stadium</size>"
+"<size=25>② 【7日目】チーム競技場でウイニング報酬を5回獲得しよう","<size=25>②  [Day 7] Get a winning reward 5 times in the team stadium</size>"
+"<size=25>② 【全達成報酬】7日目の限定ミッションを全てクリアしよう","<size=25>②  [Day 7] Complete all Day 7 missions</size>"
+"<size=25>② 【全達成報酬】1～7日目の限定ミッションを全てクリアしよう","<size=25>②  [Series] Complete all missions</size>"
```

Comment:
Did you test these? How many of them are still cut off with `<size=25>`?

Reply:
Also you have to remove the `<size=25>` markup from the japanese text (and keep them intact), otherwise the tool won't find the text to replace